### PR TITLE
feat(route): 增加机器之心首页

### PR DESCRIPTION
**Pull request by ovo-Tim**
## Example for the Proposed Route(s) / 路由地址示例

```routes
/jiqizhixin/index
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明
这个一次性只能抓10篇文章，后面看看基于 `Puppeteer` 的方案，应该能抓更多。
image

“解析全文”功能可用：
image

### CODE REVIEW

File: lib/routes/jiqizhixin/index.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                const dateTag = $(article).find('time');
+
+                return {
+                    title: titleTag.text().trim(),
+                    link: titleTag.attr('href') ? baseUrl + titleTag.attr('href') : undefined,
+                    description: summaryTag.text().trim(),
+                    pubDate: parseDate(dateTag.attr('datetime')!),
+                };
+            })
+            .toArray();
```
Comment (TonyRL) [inline lib/routes/jiqizhixin/index.ts:37]: Please use the GraphQL API instead.

File: lib/routes/jiqizhixin/index.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    url: 'jiqizhixin.com',
+    maintainers: ['ovo-tim'],
+    example: '/jiqizhixin/index',
+    description: '通过请求 HTML 获取机器之心首页，可获取10条',
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    handler: async (_) => {
+        const baseUrl = 'https://www.jiqizhixin.com';
+        const response = await ofetch(baseUrl, {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3',
```
Comment (TonyRL) [inline lib/routes/jiqizhixin/index.ts:18]: RSSHub does have a user agent of [Chrome of macOS](https://github.com/DIYgod/RSSHub/blob/26993f9ebfd917ee42a1c1574b316c614abed2bd/lib/config.ts#L380). Are there any specific reasons to use this fixed string of old Chrome on Windows?

File: lib/routes/jiqizhixin/index.ts (hunk 1 above)
Comment (ovo-Tim) [reply lib/routes/jiqizhixin/index.ts:37]: Thanks for review. But I have checked again and found that there is no specific API that can be used to capture the information.

Comment (TonyRL) [reply lib/routes/jiqizhixin/index.ts:37]: <details>
<summary>Screenshot</summary>

Screenshot


</details>

Comment (ovo-Tim) [reply lib/routes/jiqizhixin/index.ts:37]: That's strange, I got nothing here.
image

File: lib/routes/jiqizhixin/index.ts (hunk 2 above)
Comment (ovo-Tim) [reply lib/routes/jiqizhixin/index.ts]: Fixed.

File: lib/routes/jiqizhixin/index.ts (hunk 1 above)
Comment (TonyRL) [reply lib/routes/jiqizhixin/index.ts]: Try scrolling down the page

Comment (ovo-Tim) [reply lib/routes/jiqizhixin/index.ts]: Oh, Found that.

Comment (ovo-Tim) [reply lib/routes/jiqizhixin/index.ts]: The API continues to return an error code 503 even after I have obtained the CSRF token. It appears that we might need to consider using Puppeteer.

Comment (ovo-Tim) [reply lib/routes/jiqizhixin/index.ts]: I still can't make it work, it just keeps returning 503. :disappointed:

**Comment by ovo-Tim** [comment]
Very strange, I can't select unfold button by neither `page.$()`, `page.waitForSelector('.u-loadmore')` nor `waitForFunction("document.querySelector('.u-loadmore') !== null")`.

**Comment by ovo-Tim** [comment]
Sadly, I found their have offical RSS: https://www.jiqizhixin.com/rss

That means I have wasted my afernoon.

**Pull request closed without merging**